src/treq/test/test_response.py

```diff
@@ -98,3 +99,18 @@ def test_history_notimplemented(self):
 
     if not skip_history:
         test_history_notimplemented.skip = "History supported."
+
+    def test_fail_for_status_ok(self):
+        original = FakeResponse(200, Headers(), body=[b''])
```

Comment:
Using the symbolic constants from Twisted would be nice - `twisted.web.http.OK` instead of `200`, etc.